src/include/catalog/pg_proc.dat

```diff
@@ -11027,25 +11027,25 @@
   proname => 'gp_percentile_cont_float8_transition',
   proisstrict => 'f',
   prorettype => 'float8',
-  proargtypes => 'float8 float8 int8 int8',
+  proargtypes => 'float8 float8 float8 int8 int8',
```

Comment:
As was stated in the task, in 8X the patch changes the catalog, which is unacceptable in 7X!